UI/html/update.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+        <meta name="viewport" content="width= device-width, initial-scale=1.0">
+    <title>update</title>
+    <link rel="stylesheet" href="../css/main.css">
```

Comment:
Why do you work on the HTML files in a backend PR? you should raise a different PR if you want to make changes on the frontend.